osf/metrics/reports.py

```diff
@@ -285,3 +285,22 @@ class InstitutionMonthlySummaryReport(MonthlyReport):
     public_file_count = metrics.Long()
     monthly_logged_in_user_count = metrics.Long()
     monthly_active_user_count = metrics.Long()
+
+
+class PublicItemUsageReport(MonthlyReport):
+    UNIQUE_TOGETHER_FIELDS = ('report_yearmonth', 'item_osfid')
+
+    # where noted, fields correspond to defined terms from COUNTER
+    # https://cop5.projectcounter.org/en/5.1/appendices/a-glossary-of-terms.html
```

Comment:
will add a link to the research-data code of practice too -- the two share the same breakdown of total-investigations / total-requests / unique-investigations / unique-requests, which seems like the important thing to get correct here, and since not everything on osf is a "dataset" i think it'll be less confusing to use the "item" terminology

Reply:
My issue is more with invoking COUNTER compliance without Product approval then whether the definitions are exactly correct. It might be good to write something like "These fields roughly correspond to..." or "These terms should be able to correspond to COUNTER" or "Potentially these will become COUNTER fields, ensure they follow glossary terms the best you can." So they warned me against include anything concrete about COUNTER before we evaluate it more, but the verbiage  should acknowledge the "`(?)`" nature of some of these definitions and not imply they are already part of COUNTER for sure.